policyengine_us/parameters/gov/states/mi/tax/income/deductions/standard/amount.yaml

```diff
@@ -0,0 +1,56 @@
+description: Michigan provides a standard deduction of this amount if you or your spouse are born after 1946 and has reached the age of 67.
+metadata:
+  label: MI standard deduction for born after 1946 and has reached the age of 67
+  period: year
+  unit: currency-USD
+  breakdown:
+    - filing_status
+  reference:
+    - title: Michigan Standard Deduction
+      href: https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits/michigan-standard-deduction
+    - title: Michigan 2022 MI-1040 Individual Income Tax FORMS AND INSTRUCTIONS
+      href: https://www.michigan.gov/taxes/-/media/Project/Websites/taxes/Forms/2022/2022-IIT-Forms/BOOK_MI-1040.pdf?rev=86a928564e3f42449c531309673f5da7&hash=7147C48E7C9B1B8171B72DC34A66642A
+    - title: Michigan Retirement and Pension Benefits
+      href: https://www.michigan.gov/taxes/iit/retirement-and-pension-benefits
+    - title: 2021 Michigan Taxpayer Assistance Manual
+      href: https://www.michigan.gov/-/media/Project/Websites/taxes/2022RM/UNCAT/2019_Taxpayer_Assistance_Manual.pdf?rev=3a1167cc5dcf4340a4b1e1a8fe76078c
+
+SINGLE:
+  2022-01-01: 20_000
+  2021-01-01: 20_000
+  2020-01-01: 20_000
+  2019-01-01: 20_000
+  2018-01-01: 20_000
+  2017-01-01: 20_000
```

Comment:
If the value stays the same we can enter just the earliest date and leave the others out as it will automatically be set:
2017-01-01: 20_000

Reply:
same thing for other yaml files